src/components/GiveProject/GrantCard.tsx

```diff
@@ -94,16 +83,6 @@ export default function GrantCard({ grant, mode }: GrantDetailsProps) {
   // See: https://stackoverflow.com/a/66753532
   const dispatch = useAppDispatch();
 
-  const svgFillColour: string = theme.palette.type === "light" ? "black" : "white";
-
-  useEffect(() => {
-    const items = document.getElementsByClassName("project-container");
```

Comment:
why is this no longer needed? does switching to the grid component mean that when we navigate from the causes or grant list to a specific one's page it no longer preserves the scroll-height you were at on the list page?

Reply:
I thought it was no longer relevant, due to the removal of the project-container class.

What is this trying to achieve? Profile -> list, scroll to top? Or list -> profile, scroll to top?